azure-pipelines/build.yml

```diff
@@ -49,6 +56,22 @@ steps:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: '$(Agent.TempDirectory)/coverlet/reports/Cobertura.xml'
 
+- task: PowerShell@2
```

Comment:
>task: PowerShell@2 [](http://example.com/codeflow?start=2&length=18)

if you built your nupkg from a csproj you wouldn't need this task at all

Reply:
Commented: https://github.com/microsoft/sqltoolsservice/pull/1747#discussion_r1014343548
Given that, you must be signing your binaries from output folder and not a separate publish directory. In this case its separate, so signed DLLs are not consumed by csproj when packing.